### ISSUE TITLE
Fix for DNS RR in Swarm Mode causes loss of functionality

### ISSUE THREAD

**Issue by bastianeicher** [containous/traefik#2793]
### Do you want to request a *feature* or report a *bug*?

Report an issue/bug.

### What did you do?

We use Traefik on Docker Swarm to expose hundreds of distinct services. Since Docker Swarm's overlay networks get slow with Virtual IPs on this scale we have set all the relevant services to `endpoint_mode: dns_rr`. See moby/moby#30820.

We are aware that Traefik does not (yet) support load balancing with DNS RR on Swarm. This is OK for us since our target services currently all have `scale: 1`. Up until Traefik 1.4.x this worked just fine.

### What did you expect to see?

Traefik 1.5 continues to discover and expose services with `endpoint_mode: dns_rr` even if it does not support load balancing or displaying the appropriate IPs in the Web UI (yet).

### What did you see instead?

Traefik 1.5 ignores all services with `endpoint_mode: dns_rr`. This is probably caused by #2490.

While this fix prevents people from accidentally relying non-functional DNS RR load-balancing it also blocks a previously working scenario.

### Output of `traefik version`: (_What version of Traefik are you using?_)

```
Version:      v1.5.0
Codename:     cancoillotte
Go version:   go1.9.2
Built:        2018-01-23_04:42:32PM
OS/Arch:      linux/amd64
```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

```toml
defaultEntrypoints = ["http"]
constraints = ["tag==public"]
logLevel = "WARN"

[accessLog]
format = "json"

[docker]
endpoint = "unix:///var/run/docker.sock"
watch = true
exposedbydefault = false
swarmmode = true

[web]
readOnly = true

[entryPoints.http]
address = ":80"
```

### If applicable, please paste the log output in debug mode (`--debug` switch)

```
Ignored endpoint-mode not supported, service name: myservice
```

**Comment by mmatur** [comment]
Hi @bastianeicher, 

Many thanks for your interest in the project

Could you provide us a basic compose file example to be able to reproduce your issue please ?

**Comment by traefiker** [comment]
Closed by #2795.
